impeller/entity/entity_pass.cc

```diff
@@ -620,10 +478,17 @@ EntityPass::EntityResult EntityPass::GetEntityForElement(
       return EntityPass::EntityResult::Skip();
     }
 
-    auto subpass_coverage =
-        (subpass->flood_clip_ || subpass_backdrop_filter_contents)
-            ? coverage_limit
-            : GetSubpassCoverage(*subpass, coverage_limit);
+    std::shared_ptr<FilterContents> image_filter =
+        subpass->delegate_->WithImageFilter(Rect(), subpass->transform_);
+
+    auto subpass_coverage = ComputeSaveLayerCoverage(
+        subpass->bounds_limit_.value_or(Rect::MakeMaximum()),  //
+        subpass->transform_,                                   //
+        coverage_limit.value(),                                //
+        image_filter,                                          //
+        /*flood_clip=*/!!subpass_backdrop_filter_contents      //
```

Comment:
Does this need to be true when the subpass contains something with a destructive blend too? Or is that already taken care of by info we get from the DL?

Reply:
Ah, just read the comments. It sounds like `content_coverage` incorporates this.

Reply:
Right, contents_coverage should be unbounded when it contains an entity that has destructive blend vs the saveLayer itself having a destructive blend. Mind bending 😆 , it took me a while to process this.